src/client/Microsoft.Identity.Client/TokenCache.ITokenCacheInternal.cs

```diff
@@ -1173,5 +1207,21 @@ bool ITokenCacheInternal.HasTokensNoLocks()
                 .ToList()
                 .ForEach(accItem => _accessor.DeleteAccount(accItem));
         }
+
+        // Returns whether InitiateLongRunningProcessInWebAPI was called (user assertion is specified in this case)
+        private bool InitiateLongRunningOboWasCalled(AuthenticationRequestParameters requestParameters)
+        {
+            return requestParameters.ApiId == ApiEvent.ApiIds.AcquireTokenOnBehalfOf &&
```

Comment:
Is there a way to find out via telemetry if folks call AcquireTokenOBO or this new API? Maybe we should have a different ApiId here?